Common/src/geometry_structure.cpp

```diff
@@ -16064,6 +17403,288 @@ void CPhysicalGeometry::MatchActuator_Disk(CConfig *config) {
+        
+        
+        for (iVertex = 0; iVertex < nVertex[iMarker]; iVertex++) {
+          iPoint = vertex[iMarker][iVertex]->GetNode();
+          iPointGlobal = node[iPoint]->GetGlobalIndex();
+          
+          if (node[iPoint]->GetDomain()) {
+            
+            /*--- Coordinates of the boundary point ---*/
+            
+            Coord_i = node[iPoint]->GetCoord(); mindist = 1E6; pProcessor = 0; pPoint = 0;
+            
+            /*--- Loop over all the boundaries to find the pair ---*/
+            for (iProcessor = 0; iProcessor < nProcessor; iProcessor++)
+              for (jVertex = 0; jVertex < Buffer_Receive_nVertex[iProcessor]; jVertex++) {
+                jPoint = Buffer_Receive_Point[iProcessor*MaxLocalVertex_Periodic+jVertex];
+                jPointGlobal = Buffer_Receive_GlobalIndex[iProcessor*MaxLocalVertex_Periodic+jVertex];
+                jVertex_ = Buffer_Receive_Vertex[iProcessor*MaxLocalVertex_Periodic+jVertex];
+                jMarker = Buffer_Receive_Marker[iProcessor*MaxLocalVertex_Periodic+jVertex];
+                
+                /*--- Get marker index of the periodic donor boundary. ---*/
+                if ((jPointGlobal != iPointGlobal) ) {
+                  
+                  /*--- Get the position vector from rot center to point. ---*/
+                  dx = Coord_i[0] - center[0];
+                  dy = Coord_i[1] - center[1];
+                  if (nDim == 3) {
+                    dz = Coord_i[2] - center[2];
+                  } else {
+                    dz = 0.0;
+                  }
+                  
+                  /*--- Compute transformed point coordinates. ---*/
+                  rotCoord[0] = rotMatrix[0][0]*dx
+                  + rotMatrix[0][1]*dy
+                  + rotMatrix[0][2]*dz + translation[0];
+                  
+                  rotCoord[1] = rotMatrix[1][0]*dx
+                  + rotMatrix[1][1]*dy
+                  + rotMatrix[1][2]*dz + translation[1];
+                  
+                  rotCoord[2] = rotMatrix[2][0]*dx
+                  + rotMatrix[2][1]*dy
+                  + rotMatrix[2][2]*dz + translation[2];
+                  
+                  
+                  /*--- Compute the distance ---*/
+                  
+                  dist = 0.0; for (iDim = 0; iDim < nDim; iDim++) {
+                    Coord_j[iDim] = Buffer_Receive_Coord[(iProcessor*MaxLocalVertex_Periodic+jVertex)*nDim+iDim];
+                    dist += pow(Coord_j[iDim]-rotCoord[iDim],2.0);
+                  } dist = sqrt(dist);
+                  
+                  if (((dist < mindist) && (iProcessor != rank)) ||
+                      ((dist < mindist) && (iProcessor == rank) && (jPoint != iPoint))) {
+                    mindist = dist; pProcessor = iProcessor; pPoint = jPoint; pPointGlobal = jPointGlobal;
+                    pVertex = jVertex_; pMarker = jMarker;
+                    if (dist == 0.0) break;
```

Comment:
I ended up removing that check entirely, since the cost is not prohibitive (it wasn't doing much), and we don't want to have an arbitrary tolerance that might cause us to find the wrong points if it is too large.

Reply:
It does not need to be arbitrary, it can be a fraction of the local cell sizes.